### DISCUSSION TITLE
Node minimum requrements check

### DISCUSSION THREAD

**Discussion by rioda-org** [idena-network/idena-go#673] [Ideas]
It would be great if Idena node could perform test of it's enviroment on first instalation.

**CPU+RAM** - Perform simulation of mining for few full blocks. If it's unable to produce them, notify user that his hardware is too low.

**Network/Router** - Perform simulation of large number of connections as needed during validation session. Again if unable to perform them, notify user that his network connectivity/router will cause problems.

This would hopefully urge members not to run their nodes on weak hardware and bad routers.
Hopefully as a result, we would get:
- less empty blocks
- less flips/words that can't load
- less failed Candidates

**Comment by ubiubi18** [comment]
What if the stresstest would be done on a real testnet? only accounts would become eglige to activate an invite, once they passed a test verification on a real testnet with daily 14.30 utc flipsessions, also with the requirement to post own flips?

**Comment by rioda-org** [comment]
Found this pretty good connection testing tool

http://www.dslreports.com/speedtest
